css/css-syntax-3/tokenizing-and-parsing/input-byte-stream/i18n-css-syntax-bytes-001.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html  lang="en" >
+<head>
+<meta charset="ISO-8859-15"/>
+<title>CSS3 Syntax, input byte stream: CSS with UTF-signature</title>
+<link rel='author' title='Richard Ishida' href='mailto:ishida@w3.org'>
+<link rel='help' href='http://www.w3.org/TR/css3-syntax/#input-byte-stream'>
+<script src="../../../resources/testharness.js"></script>
+<script src="../../../resources/testharnessreport.js"></script>
+<link rel="stylesheet" type="text/css" href="../../resources/../testharness.css">
```

Comment:
You should remove the references to `testharness.css` (especially because it's wrong).